### ISSUE TITLE
Make channel group card size customizable

### ISSUE THREAD

**Issue by simpleavinash** [TeamNewPipe/NewPipe#9387]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for NewPipe itself and that requests for adding a new service need to be made at [NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

I'm fan of the classis horizontal scrolling (v24.0).

### Why do you want this feature?

Make channel group card size customizable. It only fits 3 cards. I want to revert back this particular channel group list view feature. Please solve this.

### Additional information

Feedback for v24.1 on Channel group 

Dear Sir, UI or layout modifications can't be welcomed by everybody. I've been using NewPipe for years and I've become habitual to the classic layout. If you make modifications in layout, please make it optional in settings. I personally don't like this new vertical scrolling layout (List and Big card view) of channel groups, it irritates me everytime I use this feature, 10 times in a day at least.

Please make this mod optional in settings, so whoever like it will turn it ON and many like me will keep it Off. In my opinion, UI modification shouldn't be mandatory to all, only Feature or functionality should.

Thank you, Rest is perfect. 👍
(I made github account only due to this inconvenience).

**Comment by opusforlife2** [comment]
The layout follows your list/grid setting. If you toggle that to grid, you'll get the old layout.

**Comment by opusforlife2** [comment]
> doesn't stay remembered

I just mentioned:
>The layout follows your list/grid setting.

If you would like that to be changed (for the layout to be independent of the setting), you can change your title and description to state that.

**Comment by opusforlife2** [comment]
This (along with Downloads) is the only place where the list/grid toggle has been provided separately. We could make a decision on whether manually changed layouts should stop following the setting, or whether there should be toggles for the layout at all, instead of enforcing the setting everywhere.

Conversely, why should there be a list/grid setting at all, even? All views could be independently toggle-able between list and grid, and the state could be saved for each such view independent of the others.

**Comment by simpleavinash** [comment]
Dear all, Thank you for paying attention to my request.


A. This version24 layout (classic or legacy style) is my first choice. Can you please keep this layout continue to next versions. And keep this (Horizontal scrolling) style also in settings for those who like it, and make it optional to choose the new version24.1 list view.

![A](https://user-images.githubusercontent.com/117650923/201490761-813ff2d6-c479-413c-afed-c98eeb401b6d.png)

B. If you have to change the channel group to list view, and want to remove the classic group style, then please make an option to customize the size of cards. I want to fit more than 3 (say 6 or 8) cards here.

![B](https://user-images.githubusercontent.com/117650923/201491037-a434c5d1-3d08-49fb-975c-b8d8ebf7f8df.png)

**Comment by simpleavinash** [comment]
I'm currently using this https://github.com/TeamNewPipe/NewPipe/files/9942509/app-debug4.zip app from  (https://github.com/TeamNewPipe/NewPipe/pull/9182), than new official v24.1 release, Due to new channel group list layout.
My point is, I want to stick to the old card layout.

**Comment by simpleavinash** [comment]
> The layout follows your list/grid setting. If you toggle that to grid, you'll get the old layout.

Dear sir,
Old card had smaller size, while new card are very big.

**Comment by simpleavinash** [comment]
> Thanks for your feedback. What do you think about vertical scrolling? These three rows will remain static instead of one row as in case of horizontal scrolling. It'll also require less scrolling compared to classic one.
> 
> <img alt="119712128-9f825e80-bea3-11eb-89e9-01b77e9ed459" height="600" src="https://user-images.githubusercontent.com/79245641/201491558-eeba5f41-e4a7-4cd8-b2c5-967e58a11782.png">

Dear sir,
Before making it official release, I will have to use it for 1-2 days first. So please put the app for testing. But it's still a chance that, Will It be acceptable by everybody. Because there will be 50% of users who might not like it. So whenever you make any UI modification in release app, please put it as an optional in settings. Don't apply it to all users.

**Comment by simpleavinash** [comment]
@SameenAhnaf I was not active on this forum, I was a normal app user, just a week ago. I came here only after I got the new release.
Thank you for your kind words.

**Comment by somos61** [comment]
> Dear all, Thank you for paying attention to my request.
> 
> A. This version24 layout (classic or legacy style) is my first choice. Can you please keep this layout continue to next versions. And keep this (Horizontal scrolling) style also in settings for those who like it, and make it optional to choose the new version24.1 list view.
> 
> <img alt="A" height="600" src="https://user-images.githubusercontent.com/117650923/201490761-813ff2d6-c479-413c-afed-c98eeb401b6d.png">
> 
> B. If you have to change the channel group to list view, and want to remove the classic group style, then please make an option to customize the size of cards. I want to fit more than 3 (say 6 or 8) cards here.
> 
> <img alt="B" height="600" src="https://user-images.githubusercontent.com/117650923/201491037-a434c5d1-3d08-49fb-975c-b8d8ebf7f8df.png">

Let me join this exact request. Please, keep channel group old style layout (or make it optional in settings at least)

**Comment by FoxSays** [comment]
I would also like there to be an option to control this behavior. At the very least, it should not be tied to the list/grid setting as it controls a very different type of user interface element.
